ocs_ci/templates/ocs-deployment/local-volume.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: local.storage.openshift.io/v1
 kind: LocalVolume
 metadata:
   name: local-block
-  namespace: local-storage
+  namespace: openshift-local-storage
```

Comment:
@clacroix12 is this local-storage namespace suppose still be called local-storage for OCP version < 4.6 ? If so, how do we change it?  Or are we OK to use this new namespace also for older versions of OCP?

Reply:
@petr-balogh I commented about this on #2640 and wanted to test using the new namespace for OCP < 4.6. The previous PR test was OCS 4.5 on OCP 4.5 and it passed so I do not think we will run into any deployment issues if we are using the new namespace. As long as anything that needs to reference the local storage namespace is doing so through the constant we have defined, we should be good to go.

Reply:
@clacroix12 only one issue I can think of is running against manually deployed cluster on OCP 4.5. Then if we somewhere query in that namespace, we will be pissed of ;)

Also was thinking about upgrade of such 4.5 cluster to 4.6 but for this kind of solution that we will use the same namespace and if we install directly to this namespace also 4.5 we should be ok to run post upgrade as we will use the same namespace.

So only one issue I see is with manual deployment but not sure how to solve this.  Some dynamic assignment of value based of current state could be the solution and store the value somewhere in the config.ENV section with our default value overwritten by old namespace if it's found and used .  But let's try to not overcomplicate the things now and see how it will go.  You can at least document this somewhere as known issue.

Reply:
is this resolved now for this PR to be merged?

Reply:
This should be handled (or at least able to be worked around) through changing the local storage namespace from a constant to a configuration variable. Resolving.